apps/condo/domains/organization/components/DeleteEmployeeButtonWithReassignmentModel.tsx

```diff
@@ -0,0 +1,245 @@
+import { useGetOrganizationEmployeeTicketsForReassignLazyQuery, useUpdateTicketsForReassignmentEmployeeMutation } from '@app/condo/gql'
+import { OrganizationEmployee, Ticket } from '@app/condo/schema'
+import { notification, Row } from 'antd'
+import isEmpty from 'lodash/isEmpty'
+import React, { useCallback, useMemo, useState } from 'react'
+
+import { IUseSoftDeleteActionType } from '@open-condo/codegen/generate.hooks'
+import { ArrowDownUp } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { Alert, Button, Modal, Space, Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import { GraphQlSearchInput } from '@condo/domains/common/components/GraphQlSearchInput'
+import { runMutation } from '@condo/domains/common/utils/mutations.utils'
+import { sleep } from '@condo/domains/common/utils/sleep'
+import { searchEmployeeUser } from '@condo/domains/ticket/utils/clientSchema/search'
+
+
+interface IDeleteEmployeeButtonWithReassignmentModel {
+    buttonContent: string
+    softDeleteAction: IUseSoftDeleteActionType<OrganizationEmployee>
+    disabled?: boolean
+    employee: OrganizationEmployee
+    activeTicketsOrganizationEmployeeCount: number
+}
+
+export const DeleteEmployeeButtonWithReassignmentModel: React.FC<IDeleteEmployeeButtonWithReassignmentModel> = ({
+    buttonContent,
+    softDeleteAction,
+    disabled = false,
+    employee,
+    activeTicketsOrganizationEmployeeCount,
+}) => {
+    const intl = useIntl()
+    const ConfirmReassignEmployeeTitle = intl.formatMessage({ id: 'employee.reassignTickets.title' })
+    const ConfirmDeleteButtonLabel = intl.formatMessage({ id: 'employee.reassignTickets.buttonContent.deleteUser' })
+    const ConfirmReassignTicketsButtonLabel = intl.formatMessage({ id: 'employee.reassignTickets.buttonContent.reassignTickets' })
+    const SearchPlaceholderLabel = intl.formatMessage({ id: 'EmployeesName' })
+    const AlertTitleLabel = intl.formatMessage({ id: 'employee.reassignTickets.alert.title' }, { activeTicketsOrganizationEmployeeCount, employeeName: employee?.name })
```

Comment:
```suggestion
    const AlertTitleLabel = intl.formatMessage({ id: 'employee.reassignTickets.alert.title' }, { activeTicketsOrganizationEmployeeCount, employeeName: employee?.name || null })
```
or employee?.name || ''